### DISCUSSION TITLE
【Move 0x04】操作资源

### DISCUSSION THREAD

**Discussion by 99Kies** [NonceGeek/Move-Camp#118] [Projects-Others]
**教程见：**

https://github.com/WeLightProject/Web3-dApp-Camp/blob/main/move-dapp/CryptoXiuXian%20in%20Move%20Lang.md#144-%E6%93%8D%E4%BD%9C%E8%B5%84%E6%BA%90

**回复截图打卡**

截图任务一

![image](https://user-images.githubusercontent.com/45918704/181919298-d3387f21-3585-40bb-a757-233137d04e23.png)

截图任务二

![image](https://user-images.githubusercontent.com/45918704/181919309-2c9b9b1d-229c-4755-a6ad-1ff9be0aee04.png)

截图任务三

![image](https://user-images.githubusercontent.com/45918704/181919323-2feabf62-1c9f-4c37-ad6c-f0dcdcbae59b.png)

**Comment by qiwihui** [comment]
It's me again!

![Screen Shot 2022-07-31 at 10 15 50 AM](https://user-images.githubusercontent.com/3297411/182007009-1c9f53ea-d51c-45a2-8f0c-4bb28850cc6d.png)
![Screen Shot 2022-07-31 at 10 17 40 AM](https://user-images.githubusercontent.com/3297411/182007010-44bac4c6-41bf-4994-8b1d-a2ca99522ea7.png)
![Screen Shot 2022-07-31 at 10 21 37 AM](https://user-images.githubusercontent.com/3297411/182007013-14608b30-632b-4647-b9f4-445d57603cbe.png)

**Reply by leeduckgo** [reply]
¥88.88

**Comment by leeduckgo** [comment]
![image](https://user-images.githubusercontent.com/12784118/182010969-7336750c-43de-411b-a071-e485f1b9ec54.png)
![image](https://user-images.githubusercontent.com/12784118/182010987-df297028-e184-429e-bf0a-71f405986a7c.png)

**Comment by 0xKT** [comment]
<img width="716" alt="image" src="https://user-images.githubusercontent.com/74288668/184544106-c88630e0-ef58-49d7-be2d-351cc0b84332.png">
<img width="1143" alt="image" src="https://user-images.githubusercontent.com/74288668/184544123-5ffab97d-842e-4b97-a33a-3b873fd1c48c.png">
<img width="1158" alt="image" src="https://user-images.githubusercontent.com/74288668/184544146-5f00d066-1ebb-4347-85f6-f9aeeb2c700c.png">

**Comment by 99Kies** [comment]
![image](https://user-images.githubusercontent.com/45918704/184786974-e349bc66-c3ef-4a1a-aec7-8ccd2406c397.png)

![image](https://user-images.githubusercontent.com/45918704/184786951-3febd5b6-339d-4571-93f1-f541b0aa9273.png)

**Comment by sharkqwy** [comment]
<img width="799" alt="截屏2022-10-17 23 38 16" src="https://user-images.githubusercontent.com/1372795/196222441-25fda96b-a867-40b5-bee4-a2874a6461a2.png">
<img width="1127" alt="截屏2022-10-17 23 41 57" src="https://user-images.githubusercontent.com/1372795/196222446-77bea89c-2164-4dca-9d24-b71e835ac56b.png">
<img width="367" alt="截屏2022-10-17 23 39 47" src="https://user-images.githubusercontent.com/1372795/196222452-7605ecc3-e270-49d1-949f-2022f8022346.png">

**Comment by zhaoge5859** [comment]
打卡
<img width="703" alt="截屏2022-11-02 11 16 28" src="https://user-images.githubusercontent.com/116735858/199388656-8ecddee9-72fb-4266-a7bb-f7d4da4b462f.png">
<img width="896" alt="截屏2022-11-02 11 25 17" src="https://user-images.githubusercontent.com/116735858/199388661-ce177119-19db-4cd1-ae8c-3217ae13b9c7.png">
<img width="1263" alt="截屏2022-11-02 11 26 48" src="https://user-images.githubusercontent.com/116735858/199388663-a0a0b295-8847-46c8-a707-53179582d37e.png">
